cargo-process.el

```diff
@@ -506,6 +537,61 @@ Requires Cargo clippy to be installed."
   (interactive)
   (cargo-process--start "Clippy" cargo-process--command-clippy))
 
+;;;###autoload
+(defun cargo-process-add (crate)
+  "Run the Cargo add command.
+With the prefix argument, modify the command's invocation.
+CRATES is the name of the crate to add.
+Cargo: This command allows you to add a dependency to a Cargo.toml manifest file."
+  (interactive (list
+				(completing-read "Crate(s) to add: " cargo-process-favorite-crates)))
+  (cargo-process--start "Add" (concat cargo-process--command-add
+									  " "
+									  crate)))
+
+;;;###autoload
+(defun cargo-process-rm (crate)
+  "Run the Cargo rm command.
+With the prefix argument, modify the command's invocation.
+CRATE is the name of the crate to remove.
+Cargo: Remove a dependency from a Cargo.toml manifest file."
+  (interactive
+   (list
+	(let ((deplist (cargo-process--get-dependencies
+					(concat
+					 (cargo-process--project-root) "Cargo.toml"))))
+	  (if deplist
```

Comment:
I would change this `if` to a `when`, and move the `message` to the bottom of this function, in the false clause of the last `if`. You will also be able to remove the `progn` entirely that way